ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-ref.yaml

```diff
@@ -1,9 +1,9 @@
 ref:
   as: openshift-microshift-infra-aws-ec2
   from_image:
-    namespace: ocp
-    name: "4.16"
-    tag: upi-installer
```

Comment:
Why change the upi-installer?

Reply:
@pmtk and I thought it might not be working because the image used in steps and logs is different, so we're trying this

Reply:
Just realized, you can not scp if using the `upi-installer`, right?

Reply:
Yeah, the ssh and scp don't work because of the UID tweaks to the image

Reply:
```
unknown user 1001750000
```